docs/source/compatibility.rst

```diff
@@ -68,6 +68,37 @@ The actual change should be done in the following steps:
 - At the major update announced in the above update, change the API.
 
 
+Experimental APIs
+-----------------
+
+Thanks to many contributors, we have introduced many new features to Chainer.
+But sometimes it was only after we released the new features that we noticed that their APIs are not appropriate.
+The objective of experimental APIs is to alleviate the design failure of them.
```

Comment:
I suggest to replace:

But sometimes it was only after we released the new features that we noticed that their APIs are not appropriate.
 The objective of experimental APIs is to alleviate the design failure of them.
Any newly added API can be marked as *experimental*.

with the following:

However, we have sometimes released new features only to later notice that their APIs are not appropriate.
The objective of experimental APIs is to avoid such issues by allowing the developer to mark any newly added API as *experimental*.